### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by JulienDelemarle**
new style and add link on the fight button

### CODE REVIEW

**Review by GeorgesFORTRY — DISMISSED** [review]
group validation of the choice to use styled-components

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "react-dom": "^17.0.2",
     "react-scripts": "4.0.3",
     "sass": "^1.32.8",
+    "styled-components": "^5.2.3",
```
Comment (LoicBrassart) [inline package.json]: Choose one or the other

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
You must choose one styling tool and stick to it across the prokect

**Review by GeorgesFORTRY — APPROVED** [review]
all files are in styled-components

File: package.json (hunk 1 above)
Comment (GeorgesFORTRY) [reply package.json]: remove scss/all files are in styled-components

File: package.json [hunk 2]
```diff
@@ -9,8 +9,9 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.0",
```
Comment (LoicBrassart) [inline package.json:12]: This shouldn't suddenly pop when I ask for styling... Just saying :p

**Review by LoicBrassart — APPROVED** [review]
I see what you did there :smirk: 
But nice job, no more Evils

File: package.json (hunk 2 above)
Comment (GeorgesFORTRY) [reply package.json:12]: oups !

**Pull request merged**